expenses/migrations/0002_alter_expenses_date.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.3 on 2022-04-11 15:25
+
```

Comment:
please don't create alter files for the expenses app

Reply:
Got it